src/Models/RaspberryPiDevice.ts

```diff
@@ -306,7 +264,7 @@ export class RaspberryPiDevice extends ContainerDeviceBase {
     };
     let raspiPassword = await vscode.window.showInputBox(raspiPasswordOption);
     if (raspiPassword === undefined) {

```

Comment:
could use ! for falsy value check.
```
if (!raspiPassword)
```

Reply:
Since password can be empty string. We use `raspiPassword === undefined` instead of `!raspiPassword` here.